app/furniture/marketplace/payment_settings_controller.rb

```diff
@@ -1,11 +1,7 @@
 class Marketplace
-  class PaymentSettingsController < FurnitureController
+  class PaymentSettingsController < Controller
     def index
       authorize(marketplace, :edit?)
     end
-
-    helper_method def marketplace
-      @marketplace ||= policy_scope(Marketplace).find(params[:marketplace_id])
-    end
```

Comment:
Nice tightening.

Reply:
Oh that's helpful to see that. When I implemented this I forgot about the existence of `Marketplace::Controller`.